Userland/Libraries/LibCore/ArgsParser.cpp

```diff
@@ -383,6 +383,8 @@ void ArgsParser::print_usage_markdown(FILE* file, StringView argv0)
 
 void ArgsParser::print_version(FILE* file)
 {
+    // FIXME: Allow applications to override version string for --version.
+    //     Especially useful for Lagom applications
```

Comment:
Normally we align trailing comments with the start of the text after the `FIXME: `:
```c++
    // FIXME: Allow applications to override version string for --version.
    //        Especially useful for Lagom applications
```